### DISCUSSION TITLE
Загрузка обложки для видео

### DISCUSSION THREAD

**Discussion by dmit25** [vknet/vk#1567] [Q&A]
Не уверен есть ли на текущий момент такой API, но крайне хотелось бы иметь возможность загружать/редактировать видео с кастомной обложкой.

Спасибо за отличную библиотеку! Если интересует, от нашей компании можем подарить лицензии на prowritingaid ;)

**Comment by ghost** [comment]
На данный момент такого метода API не существует, ни открытого ни скрытого.

**Reply by dmit25** [reply]
Эээх, жаль. Спасибо за ответ!
